recipe/meta.yaml

```diff
@@ -160,20 +204,23 @@ outputs:
       summary: Metapackage of MKL headers and libraries for developing software that uses MKL
       license: LicenseRef-ProprietaryIntel
       license_family: Proprietary
-      license_file: mkl/info/LICENSE.txt  # [win]
-      license_file: mkl/info/licenses/license.txt  # [unix]
+      license_file: mkl/info/licenses/license.txt
     test:
       commands:
         - ls -A $PREFIX/lib/*  # [unix]
         - ls -A $PREFIX/include/*  # [unix]
 
-  - name: daal
+  - name: dal
+    version: {{ daal_version }}
     script: repack.sh   # [unix]
     script: repack.bat  # [win]
     build:
-      number: {{ buildnum|int + dstbuildnum|int }}
+      number: {{ daal_buildnum|int + dstbuildnum|int }}
       binary_relocation: false
       detect_binary_files_with_prefix: false
+      skip: True                                  # [win32]
```

Comment:
It's true not only for windows but for linux. 32 bit support have been dropped in 2021.* versions